<launch>
    <node name="phonePosEst" pkg="phone_localization" type="phonePosEst.py" output="screen"/>
    <param name="vehicle_pos_topic_type" type="str" value="nav_msgs/Odometry" />
    <param name="vehicle_pos_topic_name" type="str" value="msf_core/odometry" />
</launch>
